src/components/patterns/horizontal-card.tsx

```diff
@@ -49,54 +47,72 @@ const HorizontalCard = ({
     linkAttributes["aria-labelledby"] = headerId
     delete linkAttributes["aria-label"]
   }
+  let bgColor = "bg-fog-light text-black-true"
+  let btnColor = "bg-cardinal-red text-white border-cardinal-red hocus:bg-black-true hocus:text-white"
+  let linkColor = ""
 
-  return (
-    <div className="relative" {...props} ref={ref}>
-      {fullWidth && (
-        <div className="absolute left-0 top-0 z-[-10] ml-[calc(-50vw+50%)] h-full w-screen">
-          <div className="relative h-full w-full bg-black-true" {...props}>
-            <CardSprinkles position={backgroundSprinkles} />
-          </div>
-        </div>
-      )}
-
-      {!fullWidth && (
-        <FullScreenBackground compareRef={ref} className="relative h-full w-full bg-black-true">
-          <CardSprinkles position={backgroundSprinkles} />
-        </FullScreenBackground>
-      )}
+  switch (cardBgColor) {
+    case "cardinal_red":
+      bgColor = "bg-cardinal-red text-white"
+      btnColor = "bg-white text-cardinal-red hocus:bg-black-true hocus:text-white border-white"
+      linkColor = "[&_a]:text-white hocus:[&_a]:text-black-true"
+      break
+  }
 
-      <div className="centered relative mt-[77px] pb-[7.2rem] pt-[5.8rem] leading-display text-white @container @6xl:mt-0 lg:px-80">
-        <div className="grid items-center gap-2xl @6xl:grid-cols-2">
+  return (
+    <div {...props} ref={ref} className={twMerge("relative", bgColor, props.className)}>
+      <div className="rs-p-5 relative w-full leading-display @container @6xl:centered lg:px-80">
+        <div className="grid items-center gap-2xl @6xl:grid-cols-2 @10xl:gap-[8.8rem]">
           {(image || video) && (
-            <div className="relative mt-[-135px] aspect-[16/9] w-full overflow-hidden @6xl:mt-0">
-              {image}
-              {video}
+            <div className="relative h-fit w-full">
+              <div className="relative aspect-[4/3] w-full overflow-hidden">
+                {image}
+                {video}
+              </div>
+              {caption && (
+                <div className="absolute bottom-0 z-10 w-full bg-black bg-opacity-80 p-10">
+                  <div className="text-16 font-medium leading-normal text-white">{caption}</div>
+                </div>
+              )}
             </div>
           )}
 
-          <div className="">
-            {superHeader && <span className="type-0 mb-0 font-bold leading-display underline">{superHeader}</span>}
-
-            {header && (
-              <Heading id={headerId} className={twMerge("text-32", hideHeading && "sr-only")}>
-                {header}
-              </Heading>
-            )}
+          <div>
```

Comment:
Do we need this wrapper? looks like it's unused.

Reply:
Ah, this is why the div is needed 😅 
![image](https://github.com/user-attachments/assets/05e37e7e-c0bd-49f6-a97a-6605e001b01e)

Reply:
hm. i'm still not convinced this particular div is needed, since it has a child div that wraps everything
```
<div>
<div class="...">
stuff
goes
here
</div>
</div>
```
in this case the wrapping div without any classes is just extra markup.

Reply:
unless i'm not seeing the correct matching closing div

Reply:
oh yep. nvm. It's the closing way down at the bottom.